### DISCUSSION TITLE
Series Selector DicomSeriesSelector ....

### DISCUSSION THREAD

**Discussion by dcsst47** [Project-MONAI/monai-deploy-app-sdk#414] [Q&A]
Can someone please help me in figuring out Dicom Series Selector , I do not understand why series attribute ImageType value is getting a value as  None.  Although the dicom input file has the header information as  0008,0008 (ImageType): ORIGINAL\PRIMARY\AXIAL

---- This is the output from the monai-deploy-app  when  running the bundle inference operator. 
[2023-03-30 21:40:19,469] [INFO] (root) - On attribute: 'ImageType' to match value: '['PRIMARY', 'ORIGINAL']'
[2023-03-30 21:40:19,469] [INFO] (root) -     Series attribute ImageType value: None

Sample_Rules_Text = """
{
    "selections": [
        {
            "name": "CT Series",
            "conditions": {
                "Modality": "(?i)CT",
                "ImageType": ["PRIMARY", "ORIGINAL"],
                "PhotometricInterpretation": "MONOCHROME2
            }
        }
    ]
}
"""
Can someone point me to how to set the selection rules, test them if possible, because in the monai -app framework, there is no control as the render scene is all automatic from an end user perspective.  

I need help here please.

**Comment by MMelQin** [comment]
I would suggest to remove the matching criteria on "ImageType", if it still fits the need.
It would be help to have the input DICOM instance file(s) to see why the ImageType is None. BTW, the App SDK internally uses pydicom for reading the DICOM metadata and the JSON names. Please look at the [DICOM data loader operator ](https://github.com/Project-MONAI/monai-deploy-app-sdk/blob/main/monai/deploy/operators/dicom_data_loader_operator.py)for details.
